src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperBlockRewriterTest/CreatesErrorForMalformedTagHelpersWithAttributes10.diag.txt

```diff
@@ -1 +1,3 @@
+(1,2): Error RZ1035: Missing close angle for tag helper 'p'.
 (1,2): Error RZ1034: Found a malformed 'p' tag helper. Tag helpers must have a start and end tag or be self closing.
+(1,13): Error RZ1030: TagHelper attributes must be well-formed.
```

Comment:
From what I understand these make sense to me. I got these bug fixes for free. Let me know if I am wrong.